acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -16,8 +16,8 @@ def initialize_repo_on_host(host, os, nightly)
         on host, "dpkg -i puppet5-nightly-release-$(lsb_release -sc).deb"
 
       else
-        on host, "curl -O http://apt.puppetlabs.com/puppetlabs-release-pc1-$(lsb_release -sc).deb"
-        on host, "dpkg -i puppetlabs-release-pc1-$(lsb_release -sc).deb"
+        on host, "curl -O http://apt.puppetlabs.com/puppet-release-$(lsb_release -sc).deb"
+        on host, "dpkg -i puppet-release-$(lsb_release -sc).deb"
```

Comment:
Also, there is a beaker utility for this (`install_puppetlabs_release_repo_on`), but the current version of beaker it will break for the new release package path. I have a PR to fix that in https://github.com/puppetlabs/beaker-puppet/pull/9, but you'd also need to update your beaker version for that change